WordPress/src/main/java/org/wordpress/android/ui/prefs/accountsettings/AccountSettingsFragment.kt

```diff
@@ -184,6 +188,11 @@ class AccountSettingsFragment : PreferenceFragmentLifeCycleOwner(),
         this.lifecycleScope.launchWhenStarted {
```

Comment:
Great idea.. in fact, already done w/ https://github.com/wordpress-mobile/WordPress-Android/commit/d8dea834da7b404f37d767e31e3ba7a4bec2ceeb :grin:  . Since this is to land in a feature branch, I think it's ok get it in along with the tracks PR - but let me know if you think otherwise, and I can cherry-pick it into this branch.